include/swift/AST/Attr.def

```diff
@@ -630,6 +630,20 @@ DECL_ATTR(hasAsyncAlternative, HasAsyncAlternative,
   APIStableToAdd | APIStableToRemove,
   111)
 
+CONTEXTUAL_DECL_ATTR(distributed, DistributedActor,
+  DeclModifier | OnClass | OnFunc |
+  ConcurrencyOnly | DistributedOnly |
+  ABIBreakingToAdd | ABIBreakingToRemove |
+  APIBreakingToAdd | APIBreakingToRemove,
+  112)
+
+DECL_ATTR(_distributedActorIndependent, DistributedActorIndependent,
+  OnFunc | OnVar |
+  ConcurrencyOnly | DistributedOnly | UserInaccessible |
+  ABIStableToAdd | ABIStableToRemove |
+  APIBreakingToAdd | APIBreakingToRemove,
+  113)
```

Comment:
This is only used to mark the `distributedActor.address` (and transport) as "accessible always, regardless if it is distributed remote or not.

No user defined values will ever be able to have such semantics, as which is strongly aligned with our special proxy allocation logic.